kubernetes-data-platform/templates/_helpers.tpl

```diff
@@ -118,7 +118,6 @@ Create a common label block
 */}}
 {{- define "kdp.labels" -}}
 environment: {{ .Values.global.environment }}
-chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

Comment:
This is the cause of the immutable field errors we get when upgrading this chart

Reply:
Glad you found this. Mighty confusing this morning.

Reply:
It will require one last chart delete, but that "should" be it